Packs/FeedCrowdstrikeFalconIntel/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "Crowdstrike Falcon Intel Feed",
     "description": "Tracks the activities of threat actor groups and advanced persistent threats (APTs) to understand as much as possible about their known aliases, targets, methods, and more.",
     "support": "xsoar",
-    "currentVersion": "2.0.21",
+    "currentVersion": "2.0.22",
```

Comment:
Changing the context is worth bumping the minor version, let's make it `2.1.0` (make sure to replace X_Y_Z above with it, or whatever version we'll have by the time this is merged)